### PR TITLE
creates my amazing commit

### LINKED ISSUE #16: create a new file for bnewman2 Portland, OR
- [ ] Create a branch
- [ ] Add Comments
- [ ] Open a PR
- [ ] Have a discussion
- [ ] Seek Approval
- [ ] Merge a PR

### PR DESCRIPTION
**Pull request by DataDredger**
closes #16

### CODE REVIEW

File: bnewman2-Portland-Oregon.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Portland Oregon
 
 ### Food Recomendations
+- Thirsty Lion
+- 808 Grind
```
Comment (DataDredger) [inline bnewman2-Portland-Oregon.md]: why did you add this??

File: bnewman2-Portland-Oregon.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Portland Oregon
+
+### Food Recomendations
+- Thirsty Lion
+- 808 Grind
```
Comment (juliahchen) [inline bnewman2-Portland-Oregon.md:5]: Good coffee there?

File: bnewman2-Portland-Oregon.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (atraeger9) [inline bnewman2-Portland-Oregon.md]: Get the combo here

File: bnewman2-Portland-Oregon.md (hunk 2 above)
Comment (DataDredger) [reply bnewman2-Portland-Oregon.md:5]: There is always great coffee at smaller places like Barista :coffee:

File: bnewman2-Portland-Oregon.md (hunk 3 above)
Comment (atraeger9) [inline bnewman2-Portland-Oregon.md]: :100: :surfing_man:

**Review by juliahchen — APPROVED** [review]
Love coffee ☕️ 👍
